setup.py

```diff
@@ -30,5 +30,6 @@
     extras_require={
         # as recommended by aiohttp, see http://aiohttp.readthedocs.io/en/stable/#library-installation
         'aiohttp_extras': ['aiodns', 'cchardet'],
+        'aiobravado': ['aiobravado'],
```

Comment:
Yeah, it's circular. But there is an aiobravado import in http_client.py to get the AsyncioHttpFuture class. The import is done within the function to prevent issues.

Reply:
What I forgot to add: in practice I don't expect the FULL_ASYNCIO run mode to be used outside of aiobravado, so the extras_require is just there to be "correct". aiobravado will depend on bravado-asyncio and there cannot be any missing package issues.

Reply:
this could make sense ... but it will limit the eventual usage of this outside aiobravado.
I'm fine with this but we could eventually have the abstract future defined here 
Drop this for now ... we'll reconsider it if we need to

Reply:
That could be a nice way around it; unfortunately currently the HttpFuture class is part of bravado, but since people probably don't use it directly (they just get it as a return value) we could move it out of aiobravado.